### PR TITLE
packages: Update packages to 3.0.14

### PR DESCRIPTION
**Pull request by erick0z**
- Update repository to match Clear Containers release: 3.0.14
- Bump kernel version
- Bump cc-ksm-throttler
- Bump qemu-lite
- Bump clear-containers-selinux

Some packages needed a bump since the packages for Fedora 26 were corrupt
for some unknown reason.

Fixes # 250

Signed-off-by: Erick Cardona <erick.cardona.ruiz@intel.com>

### CODE REVIEW

**Comment by jcvenegas** [comment]
Why is need to bump the kernel version?

**Comment by erick0z** [comment]
@jcvenegas I noticed, for Fedora 26 only, the publish server (aka download service) had old and wrong metadata So, when I tried to install a package I had an error message saying that the package didn't exist but the rpm was actually there. I bumped the version of some packages to generate new packages and new metadata.

**Comment by jcvenegas** [comment]
@erick0z  thanks for clarify

File: clear-containers-selinux/clear-containers-selinux.spec [hunk 1]
```diff
@@ -21,7 +21,7 @@
 # Package information
 Name:			clear-containers-selinux
 Version:		0.1
-Release:		1%{?dist}
+Release:		5.<B_CNT>
```
Comment (jcvenegas) [inline clear-containers-selinux/clear-containers-selinux.spec:24]: I am not familiar with this what it represents ?

Comment (erick0z) [reply clear-containers-selinux/clear-containers-selinux.spec:24]: <B_CNT> is needed in order to have version bumps, so new packages of the same version can be published to the download server. Without this, OBS will never publish a package that has been bumped.

Comment (erick0z) [reply clear-containers-selinux/clear-containers-selinux.spec:24]: It actually makes the `Release` field work as expected in OBS.

**Review by jodh-intel — APPROVED** [review]

**Pull request merged**